src/signature_pad.ts

```diff
@@ -37,13 +37,57 @@ export interface Options extends Partial<PointGroupOptions> {
   velocityFilterWeight?: number;
   backgroundColor?: string;
   throttle?: number;
+  documentAsEventTarget?: boolean;
```

Comment:
I don't think we should pollute the options object with items used for tests.

Reply:
I'm happy to remove this in my next commit. However, it wouldn't be able to be tested using document as EventTarget.

Reply:
You should be able to do something like `signpad['_et'] = document` to force it to be a document.

Reply:
Good tip, will try it right away.